src/daemon/http/server.rs

```diff
@@ -1185,7 +1187,7 @@ async fn api_ca_deactivate(req: Request, handle: Handle) -> RoutingResult {
         req,
         Permission::CA_DELETE,
         handle.clone(),
-        render_json_res(req.state().read().await.ca_deactivate(&handle, &actor).await)
+        render_json_res(req.state().read().await.ca_delete(&handle, &actor).await)
```

Comment:
Should this fn also be renamed to delete then too?

Reply:
ack